packages/instantsearch-ui-components/src/types/Recommend.ts

```diff
@@ -0,0 +1,27 @@
+export type RecommendClassNames = Partial<{
+  root: string;
+  title: string;
+  container: string;
+  list: string;
+  item: string;
+}>;
```

Comment:
- For class names we usually provide JSDoc to help users know what each key refers to ([example](https://github.com/algolia/instantsearch/blob/55d550eb1030a3b2091917b1a84d280a73a28a3a/packages/instantsearch-ui-components/src/components/Hits.tsx#L26-L43)).
- The type should be strict, and usage can use `Partial`.